ext/oci8/oci8_failover.c

```diff
@@ -101,30 +99,33 @@ int php_oci_disable_taf_callback(php_oci_connection *connection)
 int php_oci_register_taf_callback(php_oci_connection *connection, char *callback)
```

Comment:
Change Line 99 to:
`int php_oci_register_taf_callback(php_oci_connection *connection, zval *callback)`